WordPress/Classes/ViewRelated/NUX/LoginWPComViewController.swift

```diff
@@ -20,4 +30,71 @@ class LoginWPComViewController: SigninWPComViewController, LoginViewController {
         WPAppAnalytics.track(.twoFactorCodeRequested)
         self.performSegue(withIdentifier: .show2FA, sender: self)
     }
+
+    override func localizeControls() {
+        passwordField.placeholder = NSLocalizedString("Password", comment: "Password placeholder")
+        passwordField.accessibilityIdentifier = "Password"
+        
+        let submitButtonTitle = NSLocalizedString("Next", comment: "Title of a button. The text should be capitalized.").localizedCapitalized
+        submitButton.setTitle(submitButtonTitle, for: UIControlState())
+        submitButton.setTitle(submitButtonTitle, for: .highlighted)
+        submitButton.accessibilityIdentifier = "Log In Button"
+        
+        let forgotPasswordTitle = NSLocalizedString("Lost your password?", comment: "Title of a button. ")
+        forgotPasswordButton.setTitle(forgotPasswordTitle, for: UIControlState())
+        forgotPasswordButton.setTitle(forgotPasswordTitle, for: .highlighted)
+    }
+
+    /// Sets the text of the error label.
+    /// - Note: this should become part of LoginViewController -nh
+    ///
+    func displayError(message: String) {
+        statusLabel.text = message
```

Comment:
The super-class will use the status label to show progress messages during sign in.  We'd want to ignore those since they'd look like errors rather than progress info.  Dropping the super class inheritance would help with this I suppose.

![simulator screen shot jun 27 2017 11 32 20 am](https://user-images.githubusercontent.com/1435271/27599196-9092e512-5b2d-11e7-8975-bf9fdb57805b.png)